### PR TITLE
[SNDVOL32][MMSYS] Fix French (fr-FR) layout and translation

### PR DESCRIPTION
**Pull request by KRosUser**
## Purpose

image
image
image

## Proposed changes
image
image
image

### CODE REVIEW

**Comment by learn-more** [comment]
For whoever is gonna merge this: the commit message does not follow our style.

**Comment by KRosUser** [comment]
> For whoever is gonna merge this: the commit message does not follow our style.

It's now fixed

File: dll/cpl/mmsys/lang/en-US.rc [hunk 1]
```diff
@@ -179,7 +179,7 @@ STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUPWINDOW | WS_VISIBLE | WS_CAPTION
 CAPTION "Multichannel"
 FONT 8, "MS Shell Dlg"
 BEGIN
-    LTEXT "Description goes here.", 9506, 8, 4, 212, 24
+    LTEXT "By moving the sliders below you can adjust the speakers volume of all speakers connected to this computer.", 9506, 8, 4, 212, 24
```
Comment (binarymaster) [inline dll/cpl/mmsys/lang/en-US.rc:182]: Ok first I checked whether this description may be a dynamic label that can be different depending on drivers and such, which seems to be not the case. I have addressed it in a separate commit c4bd0db0, which has reset translations for all languages. I will rebase this PR so there is no merge conflict.

**Review by binarymaster — APPROVED** [review]
It's ok now, it would be better to have some approvals from @reactos/lang-french

**Comment by KRosUser** [comment]
> It's ok now, it would be better to have some approvals from @reactos/lang-french

I don't know who's in this group, but appart from me, @HBelusca and @SergeGautherie are 2 well known French speakers too :-)

File: dll/cpl/mmsys/lang/en-US.rc [hunk 2]
```diff
@@ -179,7 +179,7 @@ STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUPWINDOW | WS_VISIBLE | WS_CAPTION
 CAPTION "Multichannel"
 FONT 8, "MS Shell Dlg"
 BEGIN
-    LTEXT "By moving the sliders below you can adjust the volume of all speakers connected to this computer.", 9506, 8, 4, 212, 24
+    LTEXT "By moving the sliders below you can adjust the speakers volume of all speakers connected to this computer.", 9506, 8, 4, 212, 24
```
Comment (binarymaster) [inline dll/cpl/mmsys/lang/en-US.rc:182]:
```suggestion
    LTEXT "By moving the sliders below you can adjust the volume of all speakers connected to this computer.", 9506, 8, 4, 212, 24
```
Typo here

File: dll/cpl/mmsys/lang/fr-FR.rc [hunk 3]
```diff
@@ -179,39 +179,39 @@ STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUPWINDOW | WS_VISIBLE | WS_CAPTION
 CAPTION "Multi-canaux"
 FONT 8, "MS Shell Dlg"
 BEGIN
-    LTEXT "By moving the sliders below you can adjust the volume of all speakers connected to this computer.", 9506, 8, 4, 212, 24
-    LTEXT "Faible", 9473, 65, 31, 14, 8
+    LTEXT "En déplaçant les curseurs ci-dessous, vous pouvez modifier le niveau du volume de toutes les enceintes connectés à cet ordinateur.", 9506, 8, 4, 212, 36
```
Comment (zefklop) [inline dll/cpl/mmsys/lang/fr-FR.rc:182]: "des enceintes". No need to explicit "toutes"

Comment (zefklop) [reply dll/cpl/mmsys/lang/fr-FR.rc:182]: "connectées" please

**Review by zefklop — CHANGES_REQUESTED** [review]
As per comments

File: dll/cpl/mmsys/lang/fr-FR.rc (hunk 3 above)
Comment (HBelusca) [reply dll/cpl/mmsys/lang/fr-FR.rc:182]:
```suggestion
    LTEXT "En déplaçant les curseurs ci-dessous, vous pouvez modifier le niveau du volume des enceintes connectées à cet ordinateur.", 9506, 8, 4, 212, 36
```

**Comment by KRosUser** [comment]
> As per comments

Fixed. Leftover from first attempt ("haut-parleurs") then changed for "enceintes" for the sake of consistancy with other dialogs

**Review by binarymaster — APPROVED** [review]

File: dll/cpl/mmsys/lang/fr-FR.rc [hunk 4]
```diff
@@ -234,13 +234,13 @@ BEGIN
...
-    IDS_SPEAKER_CENTER "&Center:"
-    IDS_SPEAKER_BACK_LEFT "&Back left:"
-    IDS_SPEAKER_BACK_RIGHT "B&ack right:"
-    IDS_SPEAKER_LOW_FREQUENCY "Low &Frequency:"
+    IDS_SPEAKER_VOLUME "Volume des enceintes"
+    IDS_SPEAKER_LEFT "&Gauche :"
+    IDS_SPEAKER_RIGHT "&Droite :"
+    IDS_SPEAKER_CENTER "&Centre :"
+    IDS_SPEAKER_BACK_LEFT "&Arrière gauche :"
+    IDS_SPEAKER_BACK_RIGHT "A&rrière droit :"
```
Comment (SergeGautherie) [inline dll/cpl/mmsys/lang/fr-FR.rc:242]:
```suggestion
    IDS_SPEAKER_BACK_RIGHT "A&rrière droite :"
```

File: dll/cpl/mmsys/lang/fr-FR.rc [hunk 5]
```diff
@@ -179,39 +179,39 @@ STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUPWINDOW | WS_VISIBLE | WS_CAPTION
 CAPTION "Multi-canaux"
 FONT 8, "MS Shell Dlg"
 BEGIN
-    LTEXT "By moving the sliders below you can adjust the volume of all speakers connected to this computer.", 9506, 8, 4, 212, 24
-    LTEXT "Faible", 9473, 65, 31, 14, 8
+    LTEXT "En déplaçant les curseurs ci-dessous, vous pouvez modifier le niveau du volume des enceintes connectées à cet ordinateur.", 9506, 8, 4, 212, 36
```
Comment (SergeGautherie) [inline dll/cpl/mmsys/lang/fr-FR.rc:182]:
```suggestion
    LTEXT "En déplaçant les curseurs ci-dessous, vous pouvez modifier le volume des enceintes connectées à cet ordinateur.", 9506, 8, 4, 212, 36
```

File: dll/cpl/mmsys/lang/fr-FR.rc [hunk 6]
```diff
@@ -234,13 +234,13 @@ BEGIN
...
-    IDS_SPEAKER_LEFT "&Left:"
-    IDS_SPEAKER_RIGHT "&Right:"
-    IDS_SPEAKER_CENTER "&Center:"
-    IDS_SPEAKER_BACK_LEFT "&Back left:"
-    IDS_SPEAKER_BACK_RIGHT "B&ack right:"
-    IDS_SPEAKER_LOW_FREQUENCY "Low &Frequency:"
+    IDS_SPEAKER_VOLUME "Volume des enceintes"
+    IDS_SPEAKER_LEFT "&Gauche :"
+    IDS_SPEAKER_RIGHT "&Droite :"
+    IDS_SPEAKER_CENTER "&Centre :"
```
Comment (SergeGautherie) [inline dll/cpl/mmsys/lang/fr-FR.rc:240]:
```suggestion
    IDS_SPEAKER_CENTER "&Centrale :"
```

**Comment by KRosUser** [comment]
@SergeGautherie @zefklop : OK for you with the last changes ?

**Review by zefklop — APPROVED** [review]

File: dll/cpl/mmsys/lang/fr-FR.rc (hunk 6 above)
Comment (HBelusca) [reply dll/cpl/mmsys/lang/fr-FR.rc]: Hmmmm...

Comment (KRosUser) [reply dll/cpl/mmsys/lang/fr-FR.rc]: linked with "enceinte".
Suggestion from Serge makes sense to me, but "centre" was not wrong either. I let you make you mind, override if required and merge :-)

**Comment by SergeGautherie** [comment]
> OK for you with the last changes ?

Yes.

**Pull request merged**